src/Shared/Recruit.Vacancies.Client/Infrastructure/Repositories/MongoDbCommunicationRepository.cs

```diff
@@ -86,5 +86,18 @@ public Task UpdateScheduledMessagesAsSentAsync(IEnumerable<Guid> msgIds, Guid ag
                 collection.UpdateManyAsync(filter, updateDef),
                 new Context(nameof(UpdateScheduledMessagesAsSentAsync)));
         }
+
+        public Task HardDelete(DateTime dispatchDateTime)
```

Comment:
should this param be "deleteBeforeDateTime" or "earliestAlloweDispatchTime" or something of the sort

Reply:
named "deleteCommunicationMessagesStaleDateTime".